pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -161,31 +124,39 @@ Secret phrase:       innocent throw harsh wild example reflect sausage leopard l
 
 > This your key (Polkadot account). Save the secret phrase in a vault securely and never share it.
 
-Now that we have a stash account, we need to fund this account with some tokens so that it has funds to perform transactions on behalf of the collator such as transactions related to onboarding your parachain.
-
-Add the stash account to the [Polkadot Signer extension](https://polkadot.js.org/) so that you can see your account appear in the [PolkadotJs Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpaseo-rpc.dwellir.com#/accounts) UI.
```

Comment:
Why remove this part?

Reply:
Do we need it? We do everything with pop cli

Reply:
True!